test/e2e/index.test.ts

```diff
@@ -115,9 +115,9 @@ describe('e2e', () => {
     await browser.close();
   });
 
-  describe('<TrapFocus />', () => {
+  describe('<FocusTrap />', () => {
     it('should loop the tab key', async () => {
-      await renderFixture('TrapFocus/OpenTrapFocus');
```

Comment:
I guess the `fixture/TrapFocus` folder should be removed as well?

Reply:
I changed the folder name to `fixtures/FocusTrap`, I didn't delete and create a new one.